### PR TITLE
Okazaki p2 diagnosis dict 518

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: src/gdcdictionary/schemas/diagnosis.yaml [hunk 1]
```diff
@@ -18365,9 +18378,11 @@ properties:
         - Prostate
         - Rectum
         - Renal Vein
+        - Rete testis
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/diagnosis.yaml]: Capitalize, I'll change the JIRA

File: src/gdcdictionary/schemas/diagnosis.yaml [hunk 2]
```diff
@@ -18389,6 +18405,8 @@ properties:
         - Trachea
         - Transplanted kidney
         - Transverse colon
+        - Tunica albuginea
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/diagnosis.yaml]: Capitalize, I'll change the JIRA

File: src/gdcdictionary/schemas/diagnosis.yaml [hunk 3]
```diff
@@ -18389,6 +18405,8 @@ properties:
...
+        - Tunica vaginalis
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/diagnosis.yaml]: Capitalize, I'll change the JIRA

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: src/gdcdictionary/schemas/diagnosis.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/diagnosis.yaml]: updated 👍

File: src/gdcdictionary/schemas/diagnosis.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/diagnosis.yaml]: updated 👍

File: src/gdcdictionary/schemas/diagnosis.yaml (hunk 3 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/diagnosis.yaml]: updated 👍

**Review by ebusygina — APPROVED** [review]

**Review by ProfOak — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
LGTM

**Review by wteouchicago — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]
👍 lgtm!

**Pull request merged**